jsonify.js

```diff
@@ -72,14 +67,18 @@ var lf_processFile = function(p_strFile) {
         l_strFileReplace = l_strFileReplace.replace(/\n/, ' ').replace(/\r/, ' ').replace(/\"/g, "\\\"");
```

Comment:
Yes, it does. I'll probably change the variable name, since you questioned it's purpose._